testing/src/test/java/io/stargate/it/CQLTest.java

```diff
@@ -111,7 +107,7 @@ public void setup() {
     table = testName;
   }
 
-  @Before
+  @BeforeEach
```

Comment:
I am not sure if this is necessary to create a stop session after and before every test suite

Reply:
+1, this should be `@BeforeAll` and `@AfterAll`.

Reply:
It used to be `@BeforeAll`, but was [changed](https://github.com/stargate/stargate/commit/210c929d0024530be9e73d59f8cebbc312386ee0#diff-f2cee11f058387da4ff5d1b14fe11887L91-L92). I'm not sure why. @mpenick : WDYT?

Reply:
Was that this change?
https://github.com/stargate/stargate/pull/42#discussion_r491104263

Reply:
Yes, I think it was #42

Reply:
looks like we can fix this now that all the connection info is static.

Reply:
I tried doing this, but unfortunately, `CQLTest` also relies on non-static `enableAuth`, which in turn required starting the OSGi container after class-level initializers... I'd rather file a separate ticket to refactor this since it's not directly related to `ccm`